src/engine/math_util.h

```diff
@@ -110,16 +104,7 @@ void min_max_3f(f32 a, f32 b, f32 c, f32 *min, f32 *max);
 void min_max_3i(s32 a, s32 b, s32 c, s32 *min, s32 *max);
 void min_max_3s(s16 a, s16 b, s16 c, s16 *min, s16 *max);
 
-// From Wiseguy
-// Round a float to the nearest integer
-ALWAYS_INLINE s32 roundf(f32 in) {
-    f32 tmp;
-    s32 out;
-    __asm__("round.w.s %0,%1" : "=f" (tmp) : "f" (in ));
-    __asm__("mfc1      %0,%1" : "=r" (out) : "f" (tmp));
-    return out;
-}
-
+#define roundf(v) ((s32) (v))
```

Comment:
Wouldn't this always floor the value?

Reply:
You are correct, it generates a different asm. I will see what I can do about it.